src/engine/enginexfader.h

```diff
@@ -17,7 +19,11 @@ class EngineXfader {
             CSAMPLE_GAIN* gain1,
             CSAMPLE_GAIN* gain2);
 
-    static const char* kXfaderConfigKey;
+    static const QString kXfaderGroup;
+    static const QString kXfaderModeKey;
+    static const QString kXfaderCurveKey;
+    static const QString kXfaderCalibrationKey;
+    static const QString kXfaderReverseKey;
```

Comment:
These are the offenders. If you can come up with a solution that removes this static data, you should be good. Again, I recommend you simply duplicate the required `QString`s/`ConfigKey`s across both files.

Reply:
Thanks for the explanation, though I'm at a loss: I read about Construct on first Use but explanations and tutorials only refer to obejcts/pointers and classes with a constructor, which is not the case for EngineXfader.
I'll revert the commit and stick with the duplicate QStrings.